### DISCUSSION TITLE
k8s pod의 로그 문의

### DISCUSSION THREAD

**Discussion by yangsu-kim** [yoonjeong-kwon/fastcampus-kubernetes#25] [Q&A] [answered]
안녕하세요.

하나의 pod가 app container와 db container가 있고
이 pod가 온프레미스 환경에서 물리적으로 다른 node에 각각 배포 되었다고 가정했을때,

다른  node에 있는 pod 끼리 db가 공유되나요?
만약 안된다면  이런 경우는 어떻게 해결하면 좋을까요? pod를 분리해야할까요?

**Comment by yoonjeong-kwon** [comment, marked as answer]
안녕하세요. 
말씀하신 것은 결국 서로 다른 노드에서 실행 중인 파드 간에 통신이 되는가를 질문하신 것으로 보입니다.
이에 대한 답변으로는 '예'라고 할 수 있습니다.
다른 노드에서 실행 중인 pod에서 db가 실행 중인 pod ip와 포트로 통신이 되는지를 테스트해보시면 확실하게 아실 수 있습니다.
강의자료에 있는 실습을 응용해서 확인해보시기 바랍니다.
감사합니다.
